src/adapter/src/catalog/builtin_table_updates/notice.rs

```diff
@@ -133,14 +156,23 @@ impl CatalogState {
                 Action::SqlStatements(text) => Datum::String(text),
             });
             // push `message_redacted` column
-            packer.push(Datum::String(&notice.message_redacted));
+            packer.push(match notice.message_redacted.as_deref() {
+                Some(message_redacted) => Datum::String(message_redacted),
+                None => Datum::Null,
+            });
             // push `hint_redacted` column
-            packer.push(Datum::String(&notice.hint_redacted));
+            packer.push(match notice.hint_redacted.as_deref() {
+                Some(hint_redacted) => Datum::String(hint_redacted),
+                None => Datum::Null,
+            });
             // push `action_redacted` column
-            packer.push(match &notice.action_redacted {
-                Action::None => Datum::Null,
-                Action::PlainText(text) => Datum::String(text),
-                Action::SqlStatements(text) => Datum::String(text),
+            packer.push(match notice.action_redacted.as_ref() {
+                Some(action_redacted) => match action_redacted {
+                    Action::None => Datum::Null,
+                    Action::PlainText(text) => Datum::String(text),
+                    Action::SqlStatements(text) => Datum::String(text),
```

Comment:
Uncovered

Reply:
This can't be covered for the same reasons as above - we only have two notices which will always produce text that doesn't need redaction.